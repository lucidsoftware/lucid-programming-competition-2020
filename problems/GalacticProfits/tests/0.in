4
3
Dusty_Logistics
32 20 82 78
Red_Rovers
64 21 70 47
Methane_Holdings_Co
70 77 18 75
