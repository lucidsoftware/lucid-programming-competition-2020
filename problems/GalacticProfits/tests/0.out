Max: 37465
Min: -3087
